tests/formulas/paje/paje_cmg.yaml

```
@@ -120,3 +120,236 @@
     paje_base: 0.4595 * 411.92
     paje_cmg: 1.7257 * 411.92 + 355.43
     paje: (0.4595 * 411.92) + (1.7257 * 411.92 + 355.43)
+
+
+- name: Après 2023, de 3 à 6 ans, la CMG est pleine
+  period: 2023-02
+  relative_error_margin: 0.01
+  input:
+    individus:
+      parent1:
+        age: 40
+        salaire_imposable: 20000
+        activite: actif
+      parent2:
+        age: 40
+        salaire_imposable: 20000
+        activite: actif
+      enfant1:
+        age: 5
+    familles:
+      famille_0:
+        parents: [parent1, parent2]
+        enfants: [enfant1]
+        en_couple: 1t
+        af_nbenf: 1
+        gar_dom: true
+        ass_mat: 1
+    foyers_fiscaux:
+      foyer_fiscal_0:
+        declarants: [parent1, parent2]
+        personnes_a_charge: [enfant1]
+    menages:
+      menage_0:
+        personne_de_reference: parent1
+        conjoint: parent2
+        enfants: [enfant1]
+  output:
+    paje_cmg: 1673.68
+
+- name: Après 2023, de 0-3 ans la CMG est pleine
+  period: 2023-02
+  relative_error_margin: 0.01
+  input:
+    individus:
+      parent1:
+        age: 40
+        salaire_imposable: 20000
+        activite: actif
+      parent2:
+        age: 40
+        salaire_imposable: 20000
+        activite: actif
+      enfant1:
+        age: 2
+    familles:
+      famille_0:
+        parents: [parent1, parent2]
+        enfants: [enfant1]
+        en_couple: 1
+        af_nbenf: 1
+        gar_dom: true
+        ass_mat: 1
+    foyers_fiscaux:
+      foyer_fiscal_0:
+        declarants: [parent1, parent2]
+        personnes_a_charge: [enfant1]
+    menages:
+      menage_0:
+        personne_de_reference: parent1
+        conjoint: parent2
+        enfants: [enfant1]
+  output:
+    paje_cmg: 1673.68
+
+
+- name: En 2022, CMG 2022 - CMG pleine de 0 à 3
```

Regrouper avec le premier test en indiquant plusieurs enfants ?
Et typo sur le Complément de libre choix du mode de garde :
```suggestion
- name: En 2022, CMG 2022 - CMG plein de 0 à 3
```